modules/constants.lua

```diff
@@ -0,0 +1,9 @@
+CONSTANTS={}
```

Comment:
why not put all constants here? Currently this file looks redundant. Also, no need for get_constants function. Just wrap return the wrapped protected table.

Reply:
me and ophir talked and we relized that we need those constants only for the interface, so this file is not needed and should be deleted.